app/javascript/components/widget-bar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { colors } from '../lib/theme';
+
+export const CurrentWidgetBar = ({ time }) => (
+  <svg width={568 * (time / 20000)} height="4">
```

Comment:
what do you think about giving names to these numbers so they seem more on purpose?